src/install/haproxy/default-requirements.mdx

```diff
@@ -0,0 +1,10 @@
+---
+headingText: Check the compatibility and requirements
+componentType: default
+---
+
+Before installation, ensure you meet these requirements:
+
+* A New Relic account. Don't have one? [Sign up for free!](https://newrelic.com/signup) No credit card required.
```

Comment:
so generally text in a bulleted list does not receive punctuation unless it's a complete sentence. that's true for the first one, but not the other two:

- either just link to the sign up account and forego our cutesy sign up cta
- remove punctuation on all bullets

Otherwise, I'd convert the two bullet points into a full sentence which might be clunky

Reply:
pretend that the person giving you copy edit advice can consistently make her subject and verbs agree

Reply:
subjects and verbs.